crow/users/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class TimeStampedModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+class User(TimeStampedModel):
+    name                    = models.CharField(max_length=10)
+    ID                      = models.CharField(max_length=30, unique=True)
```

Comment:
id는 user_id 등 다른 FK 혹은 PK와 혼동될 우려가 있어보이네요! 사용자명을 의미하는 username으로 변경해주세요!
```suggestion
    username                      = models.CharField(max_length=30, unique=True)
```

Reply:
네